modules/update-cluster-version-object.adoc

```diff
@@ -37,32 +37,37 @@ $ oc adm upgrade --include-not-recommended
 +
 [NOTE]
 ====
-The additional `--include-not-recommended` parameter includes updates that are available but not recommended due to a known risk that applies to the cluster.
+The additional `--include-not-recommended` parameter includes updates that are available with known issues that apply to the cluster.
 ====
 +
 .Example output
 [source,terminal]
 ----
-Cluster version is 4.10.22
+Cluster version is 4.10.0-0.nightly-2021-12-23-153012
 
-Upstream is unset, so the cluster will use an appropriate default.
-Channel: fast-4.11 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10)
+Upstream: https://raw.githubusercontent.com/wking/cincinnati-graph-data/cincinnati-graph-for-targeted-edge-blocking-demo/cincinnati-graph.json
```

Comment:
I would double check that this is correct, currently there's a GitHub link to what seems to be from @wking's personal fork of something 😅

Reply:
LOL!!! You're so right. Let me fix that!

Reply:
@wking does this look right? Is this an appropriate output?

Upstream is unset, so the cluster will use an appropriate default.
Channel: fast-4.11 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10)

Reply:
that's me an in internal blog.  [This][1] is a bit better, although checking again, I see the Cluster Bot cluster was still using a custom update service.  I'll start a new thread...

[1]: Upstream is unset, so the cluster will use an appropriate default.